scripts/travis/apt-install

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+install_retry_counter=0
+max_apt_retries=5
+
+# This function loops a couple of times over apt-get, hoping to
+# avoid CI errors due to errors during apt-get
+# hashsum mismatches, DNS errors and similar things
+while true; do
+	sleep ${install_retry_counter}
+	let install_retry_counter+=1
+	if [ ${install_retry_counter} -gt ${max_apt_retries} ]; then
+		exit 1
+	fi
+	apt-get clean -qqy || continue
+	apt-get update -qqy || continue
+	apt-get install -qqy --no-install-recommends $@ && break
+
+	# In case it is a network error let's wait a bit.
```

Comment:
IMHO `sleep` would belong here as it would be in-line with the "let's wait" comment. Otherwise we have this comment detached from the actual code.

Reply:
moved the sleep to the bottom again